src/table/__tests__/columns-auto-resize.test.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
 import { render, act } from '@testing-library/react';
 import createWrapper from '../../../lib/components/test-utils/dom';
 import Table, { TableProps } from '../../../lib/components/table';
-import { fireMousedown, fireMouseMove, fireMouseup, fakeBoundingClientRect } from './utils/resize-actions';
+import { firePointerdown, firePointermove, firePointerup, fakeBoundingClientRect } from './utils/resize-actions';
```

Comment:
After changing that many tests, are you not afraid that the tests of our consumers will fail too?

How much do we need this change vs how much do you want to support teams following up their release?